[SAASINT-4344] New Integration for Klaviyo: Initial manifest without assets (#20649)
DataDog/integrations-core: klaviyo/README.md

# Klaviyo

## Overview

[Klaviyo][1] is a cloud-based email and SMS marketing automation platform supporting integrations with major eCommerce, ads, and CRM platforms.

Integrate Klaviyo with Datadog to gain insights into marketing campaign communication and track eCommerce performance based on Klaviyo events.

## Setup

Follow the instructions below to configure this integration for Klaviyo Marketing and eCommerce events.

### Configuration

> #### Install Datadog Integration in Klaviyo

Reviewer: It would be great if we could get a screenshot of the Klaviyo UI where this needs to be set up

> Within your Klaviyo account, first add the Datadog Integration.

Reviewer 2: ```suggestion
Within your Klaviyo account, first add the Datadog integration.
```

1. Log in to your [Klaviyo account][2].
2. In the left-side panel, navigate to **Integrations**.
3. Click **Add integrations**.
4. Search for Datadog and click on the tile.
5. Click **Install**. The process involves simply logging into Datadog.

> #### Install Klaviyo Integration in Datadog

Reviewer: Can we call out what scopes are required to configure this integration along with the type of account or privledges needed?

Author: Yes, added below.

> Once the above installation within Klaviyo is performed, complete the Datadog integration by clicking 

Reviewer 2: ```suggestion
After the above installation within Klaviyo is performed, complete the Datadog integration by clicking 
```

> After the above installation within Klaviyo is performed, complete the Datadog integration by clicking 

Reviewer: Will the user need to authenticate the integration on the DD tile?

> **Install Integration** which will guide you through an authorization process with Klaviyo.

Reviewer 2: ```suggestion
**Install Integration** which guides you through an authorization process with Klaviyo.
```
